core/src/modules/statusReporters.ts

```diff
@@ -414,3 +412,135 @@ function mergeMetrics(metrics: StatusMetric[]) {
 
   return mergedMetrics;
 }
+
+export namespace FinalSummaryReporters {
+  const lastRunStart = new Date(api.bootTime);
+  export namespace Sources {
+    export interface SourceData {
+      name: string;
+      profilesCreated: number;
+      profilesImported: number;
+      importsCreated: number;
+      error: string;
+    }
+    export async function getData(): Promise<Array<SourceData>> {
+      const runs = await Run.findAll({
+        where: {
+          updatedAt: { [Op.gte]: lastRunStart },
+          creatorType: "schedule",
+        },
+      });
+
+      const sources: { [id: string]: SourceData } = {};
+      for (const run of runs) {
+        let source = null;
+        const schedule = await Schedule.findById(run.creatorId);
+        if (schedule) {
+          source = await schedule.$get("source");
+        }
+
+        const currentSource = sources[source.id] || {
+          name: source.name,
+          profilesCreated: 0,
+          profilesImported: 0,
+          importsCreated: 0,
+          error: null,
+        };
+        currentSource.profilesCreated += run.profilesCreated;
+        currentSource.profilesImported += run.profilesImported;
+        currentSource.importsCreated += run.importsCreated;
+        currentSource.error = currentSource.error || run.error;
+        sources[source.id] = currentSource;
+      }
+
+      return Object.values(sources);
+    }
+  }
+
+  export namespace Profiles {
+    export interface ProfileData {
+      name: null;
+      profilesUpdated: number;
+      profilesCreated: number;
+      allProfiles: number;
+    }
+
+    export async function getData() {
+      const out: ProfileData[] = [];
+      const profilesUpdated = await Profile.count({
+        where: { updatedAt: { [Op.gt]: lastRunStart } },
+      });
+
+      const profilesCreated = await Profile.count({
+        where: { createdAt: { [Op.gt]: lastRunStart } },
+      });
+      const name = null;
+      const allProfiles = await Profile.count();
+
+      const profileData = {
+        name,
+        profilesUpdated,
+        profilesCreated,
+        allProfiles,
+      };
+      out.push(profileData);
+      return out;
+    }
+  }
+
+  export namespace Destinations {
+    export interface DestinationData {
+      name: string;
+      exportsCreated: number;
+      exportsFailed: number;
+      exportsComplete: number;
+    }
+
+    export async function getData() {
```

Comment:
Similarly, this summary will only include Destinations which had at least one Export